src/app/components/authorized/subreddit/subreddit.component.ts

```diff
@@ -1,8 +1,31 @@
 import { Component, Input } from '@angular/core';
+import { NgForm } from '@angular/forms';
+import { SubredditService } from 'src/app/services/subreddit.service';
+import { SubredditCreation } from 'src/types/subreddits';
 
 @Component({
     selector: 'app-subreddit',
     templateUrl: './subreddit.component.html',
     styleUrls: ['./subreddit.component.scss'],
 })
-export class SubredditComponent {}
+export class SubredditComponent {
```

Comment:
I think the naming is confusing, as it creates a new subreddit - should be more specific.

Reply:
New component created